video_worker/tests/test_api_communicate.py

```diff
@@ -0,0 +1,242 @@
+"""
+Api communicate tests.
+"""
+import os
+import json
+import unittest
+
+from ddt import ddt, data, unpack
+from mock import Mock, patch
+
+from video_worker.abstractions import Video
+from video_worker.api_communicate import UpdateAPIStatus
+from video_worker.config import WorkerSetup
+
+
+WS = WorkerSetup()
+
+
+@ddt
+class ApiCommunicateTest(unittest.TestCase):
+    """
+    Api Communicate test class.
+    """
+    def setUp(self):
+        if os.path.exists(WS.instance_yaml):
+            WS.run()
+        self.settings = WS.settings_dict
+        self.Video = Video(
+            veda_id='XXXXXXXX2016-V00TEST'
+        )
+        self.Video.valid = True
+        self.veda_video_status='veda status'
+        self.val_video_status='val status'
+
+    def setup_api(self, VideoObject=None, **kwargs):
+        communicate_api = UpdateAPIStatus(
+            VideoObject=VideoObject,
+            veda_video_status=kwargs.get('veda_video_status', None),
+            val_video_status=kwargs.get('val_video_status', None),
+            send_veda=kwargs.get('send_veda', True),
+            send_val=kwargs.get('send_val', True),
+        )
+
+        if communicate_api.VideoObject:
+            communicate_api.VideoObject.val_id = kwargs.get('val_id', None)
+            communicate_api.VideoObject.course_url = kwargs.get('course_url', [])
```

Comment:
Yes it's actually a list of course ids. Used in api_comunicate.py https://github.com/edx/edx-video-worker/blob/master/video_worker/api_communicate.py#L178